util/dvsim/Modes.py

```diff
@@ -443,7 +443,9 @@ def merge_global_opts(tests, global_pre_build_cmds, global_post_build_cmds,
                 processed_build_modes.append(test.build_mode.name)
             test.pre_run_cmds.extend(global_pre_run_cmds)
             test.post_run_cmds.extend(global_post_run_cmds)
-            test.run_opts.extend(global_run_opts)
+            for opt in global_run_opts:
```

Comment:
This is fine, but a better long term fix would be to make all `*_opts` a set instead, so that duplicated ones are eliminated. Would you please file an issue so that it is addressed properly?

Reply:
Done: https://github.com/lowRISC/opentitan/issues/14004